File: java_and_python_assignment/average.py
# PSEUDOCODE
# 1. Input 3 scores
# 2. Compute average = (score1 + score2 + score3) / 3
# 3. If average between 90 and 100  (A)
#    If between 80 and 89  (B)
#    If between 70 and 79  (C)
#    If between 60 and 69  (D)
#    If less than 60  (F)
# 4. Print letter grade




score1 = int(input("Enter score 1: "))
score2 = int(input("Enter score 2: "))
score3 = int(input("Enter score 3: "))

avg = (score1 + score2 + score3) / 3

if 90 <= avg <= 100:
    grade = 'A'
elif 80 <= avg < 90:
    grade = 'B'
elif 70 <= avg < 80:
    grade = 'C'
elif 60 <= avg < 70:
    grade = 'D'
else:
    grade = 'F'

print("Average:", avg)
print("Grade:", grade)

File: java_and_python_assignment/Average.java
import java.util.Scanner;

public class Average {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Enter score 1: ");
        int score1 = input.nextInt();

        System.out.print("Enter score 2: ");
        int score2 = input.nextInt();

        System.out.print("Enter score 3: ");
        int score3 = input.nextInt();

        double avg = (score1 + score2 + score3) / 3.0;

        if (avg >= 90) grade = 'A';
        else if (avg >= 80) {
        System.out.print("B")
};
        else if (avg >= 70) {
        System.out.print("C")
};
        else if (avg >= 60) {
        System.out.print("D")
};
        else {
        System.out.print("F")
};

        System.out.println("Average: " + avg);
        System.out.println("Grade: " + grade);
}
}
